module/platform/Webots/src/Webots.cpp

```diff
@@ -254,6 +254,11 @@ namespace module::platform {
                 if (duration.count() > 0) {
                     speed = diff / (double(duration.count()) / double(NUClear::clock::period::den));
                 }
+                else {
+                    // We have a speed of 0, which means 'move as fast as you can'
+                    // 5.236 == 50 rpm which is similar to the max speed of the servos
+                    speed = 5.236;
```

Comment:
this seems strange. 
Basically if the clock hasn't progressed, we set the servo to move as quickly as possible??

Reply:
If the servo was meant to get to its target now or before now, then we make it move as quick as possible.

Reply:
That makes a lot more sense. Could you please update the comment as such? That comment as is made no sense to me without that context

Reply:
How is this?

Reply:
Perfect